tests/Microsoft.DotNet.XHarness.Tests.Runners.Tests/xUnit/XUnitFilterTests.cs

```diff
@@ -0,0 +1,142 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This is great! Will help me debug the TestRunner for sure.

Before, I was creating a new command line project to instantiate the TestRunner

Reply:
I have a plan to do something more interesting for this :)